pmm-app/src/pmm-qan/panel/components/Details/Details.tsx

```diff
@@ -61,48 +62,59 @@ export const DetailsSection: FC = () => {
   useEffect(() => setLoadingDetails(loading || metricsLoading), [loading, metricsLoading]);
 
   return (
-    <div className={cx(styles.detailsGrid, 'query-analytics-details')} data-qa="query-analytics-details">
-      <div className="details-tabs">
-        <Divider className={styles.zeroMargin} />
-        <Tabs
-          activeKey={activeTab}
-          onChange={(newTab) => {
-            changeActiveTab(newTab);
-            setActiveTab(newTab);
-          }}
-          tabPosition="top"
-          destroyInactiveTabPane
-          tabBarExtraContent={(
-            <Button type="default" size="small" onClick={closeDetails}>
-              {Messages.closeDetails}
-            </Button>
-          )}
-        >
-          <TabPane tab={<span>{Messages.tabs.details.tab}</span>} key={TabKeys.details}>
-            <Metrics databaseType={databaseType} totals={totals} metrics={metrics} loading={metricsLoading} />
-          </TabPane>
-          {showExamplesTab ? (
-            <TabPane tab={<span>{Messages.tabs.examples.tab}</span>} key={TabKeys.examples}>
-              <Example
-                fingerprint={fingerprint}
+    <Scrollbar className={styles.scrollArea}>
+      <div className={cx(styles.detailsGrid, 'query-analytics-details')} data-qa="query-analytics-details">
```

Comment:
shouldn't we get rid of hard coded class name and move antd style to emotion?

Reply:
Yes, ofc, It will be fully done in PMM-6200